<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectoriesPage</class>
 <widget class="QWidget" name="DirectoriesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>278</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="incomingGBox">
     <property name="title">
      <string>Incoming Directory</string>
     </property>
     <layout class="QGridLayout" name="incomingGBoxGLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="incomingDir">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="incomingButton">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Browse</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/images/directoryselect_24x24_shadow.png</normaloff>:/images/directoryselect_24x24_shadow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="partialsGBox">
     <property name="title">
      <string>Partials Directory</string>
     </property>
     <layout class="QGridLayout" name="partialsGBoxGLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="partialsDir">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="partialButton">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Browse</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/images/directoryselect_24x24_shadow.png</normaloff>:/images/directoryselect_24x24_shadow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="sharedGBox">
     <property name="title">
      <string>Shared Directories</string>
     </property>
     <layout class="QVBoxLayout" name="sharedGBoxVLayout">
      <item>
       <layout class="QHBoxLayout" name="shareDownloadDirectoryHLayout">
        <item>
         <widget class="QCheckBox" name="shareDownloadDirectoryCB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Automatically share incoming directory (Recommended)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editShareButton">
          <property name="text">
           <string>Edit Share</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="rememberHashesHLayout">
        <item>
         <widget class="QCheckBox" name="rememberHashesCB">
          <property name="toolTip">
           <string>Remember file hashes even if not shared. 
This might be useful if you're sharing an 
external HD, to avoid re-hashing files when 
you plug it in.</string>
          </property>
          <property name="text">
           <string>Remember hashed files for </string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rememberHashesSB">
          <property name="suffix">
           <string> days</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cleanHashCachePB">
          <property name="toolTip">
           <string>Forget any hashed file that is not anymore shared.</string>
          </property>
          <property name="text">
           <string>Clean Hash Cache</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="autoCheckDirectoriesHLayout">
        <item>
         <widget class="QCheckBox" name="autoCheckDirectories_CB">
          <property name="text">
           <string>Auto-check shared directories every </string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autoCheckDirectoriesDelay_SB">
          <property name="suffix">
           <string> minute(s)</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="mainVSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>shareDownloadDirectoryCB</tabstop>
  <tabstop>editShareButton</tabstop>
  <tabstop>rememberHashesCB</tabstop>
  <tabstop>rememberHashesSB</tabstop>
  <tabstop>cleanHashCachePB</tabstop>
  <tabstop>autoCheckDirectories_CB</tabstop>
  <tabstop>autoCheckDirectoriesDelay_SB</tabstop>
  <tabstop>incomingDir</tabstop>
  <tabstop>incomingButton</tabstop>
  <tabstop>partialsDir</tabstop>
  <tabstop>partialButton</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
